static/openapi/vwrs-spec.json

```diff
@@ -1164,6 +1228,194 @@
+          },
+          {
+            "in": "query",
+            "name": "created_at",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The creation timestamp when user first time entered the waiting room."
+          },
+          {
+            "in": "query",
+            "name": "client_version",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The version of the EdgeWorker."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully fetched the user's position status.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GetStatusResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request."
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/vwr/v1/position/{domain_key}": {
+      "get": {
+        "summary": "Provide initial position in the waiting room",
+        "description": "Provide initial position in the waiting room.",
+        "tags": [
+          "Position"
+        ],
+        "security": [
+          {
+            "APIKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "domain_key",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique key of a domain."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully obtained waiting room data.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GetAccessResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request."
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/vwr/v1/position/{domain_key}/{visitor_id}": {
+      "get": {
+        "summary": "Provide current position in the waiting room",
+        "description": "Provides the visitor's position in the waiting room.\\n\\nA position of 0 indicates that the visitor is no longer in the waiting room and has gained access to the requested resource.",
```

Comment:
And here's that third one. Are we talking about clients, users, or visitors? Pick one and stick with it throughout.

Reply:
This should be user's.